README.md

```diff
@@ -70,10 +70,6 @@ Return the code unit index of the most recently discovered break position, as an
 
 The `breakType` of the most recently discovered segment. If there is no current segment (e.g., a just-instantiated SegmentIterator, or one which has reached the end), or if the break type is "grapheme", then this will be `undefined`.
 
-For most programmers, the most important differences may be
```

Comment:
Removing this information from this point would probably make more sense if we merge the other PR. Also, I think the bulleted description is more clear than a big paragraph, so I don't understand why that change was made.

Reply:
I was trying to move the details of `breakType` out of this specific getter. The latest update dedicates a specific section to the iteration result fields.